### PR TITLE
Update quickstart config section

### PR DESCRIPTION
**Pull request by chasefleming**
Addresses: https://github.com/onflow/fcl-js/issues/1696

### CODE REVIEW

File: docs/tutorials/flow-app-quickstart.mdx [hunk 1]
```diff
@@ -4,14 +4,18 @@
 
 **Last Updated:** January 11th 2022
 
-> **Note**: This page will walk you through a very bare bones project to get started building a web3 dapp using the Flow Client Library (FCL). If you are looking for a clonable repo, Flow community members have created quickstart templates for different JavaScript frameworks (e.g. [Next.js](https://github.com/muttoni/fcl-nextjs-quickstart), [SvelteKit](https://github.com/muttoni/fcl-sveltekit-quickstart), [Nuxt](https://github.com/bluesign/nuxt3-fcl)). You can consult the complete list [here](https://github.com/ph0ph0/Get-The-Flow-Down#fcl).
+> **Note**: This page will walk you through a very bare bones project to get started building a web3 dapp using the Flow Client Library (FCL). If you are looking for a clonable repo, check out the [scaffolds availabe in the Flow CLI](https://developers.flow.com/tooling/flow-cli/super-commands#using-scaffolds).
```
Comment (nialexsan) [inline docs/tutorials/flow-app-quickstart.mdx:7]: switch to relative links

File: docs/tutorials/flow-app-quickstart.mdx [hunk 2]
```diff
@@ -4,14 +4,18 @@
...
 
 ## Video Walkthrough
 
 Below is a video walkthrough covering the contents of this tutorial. You are welcome to watch or skip the video walkthrough depending on your style of learning.
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/uEoh9SnjqCk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
+## Before You Start
+
+Make sure you have Flow CLI installed. If you don't, follow the [installation instructions](https://developers.flow.com/tooling/flow-cli/install).
```
Comment (nialexsan) [inline docs/tutorials/flow-app-quickstart.mdx:17]: switch to relative links

File: docs/tutorials/flow-app-quickstart.mdx [hunk 3]
```diff
@@ -38,6 +42,44 @@ npx create-next-app@latest flow-app
...
+
+### Setting up Flow
+
+Then, using the Flow CLI, create a new `flow.json` file in the root of your app. This file will contain the Flow configuration for your project.
+
+```sh
+flow init
+```
+
+We don't recommend keeping private keys in your `flow.json`, so for good practice let's move our private key to a `emulator.key` file in the root of our project and point to it using the [key/location pattern](https://developers.flow.com/tooling/flow-cli/flow.json/security#private-account-configuration-file) used by Flow CLI. This file will be ignored by git, so it won't be committed to your repository.
```
Comment (nialexsan) [inline docs/tutorials/flow-app-quickstart.mdx:55]: switch to relative links

File: docs/tutorials/flow-app-quickstart.mdx [hunk 4]
```diff
@@ -167,21 +205,40 @@ You should now be able to log in or sign up a user and unauthenticate them. Upon
...
+One of the main things you'll often need to do when building a dapp is query the Flow blockchain and the smart contracts deployed on it for data. Since smart contracts will live on both Testnet and Mainnet, let's put the account address where the smart contract lives into the `flow.json` so FCL can pick it up. We are going to also use an alias for testnet.
 
-> **Replace file:** `./flow/config.js`
+If you want to see how to use local files for emulator and learn more about aliases, check out the [flow.json documentation](https://developers.flow.com/tooling/flow-cli/flow.json/configuration). For now, know that an alias lets us specify a contract address based on a specific network (e.g. testnet/mainnet).
```
Comment (nialexsan) [inline docs/tutorials/flow-app-quickstart.mdx:210]: relative links

**Review by nialexsan — COMMENTED** [review]
a few suggestions

File: docs/tutorials/flow-app-quickstart.mdx (hunk 1 above)
Comment (chasefleming) [reply docs/tutorials/flow-app-quickstart.mdx:7]: @nialexsan done

Comment (nialexsan) [reply docs/tutorials/flow-app-quickstart.mdx:7]: bump

Comment (nialexsan) [inline docs/tutorials/flow-app-quickstart.mdx:7]:
```suggestion
> **Note**: This page will walk you through a very bare bones project to get started building a web3 dapp using the Flow Client Library (FCL). If you are looking for a clonable repo, check out the [scaffolds availabe in the Flow CLI](../tooling/flow-cli/super-commands.md#using-scaffolds).
```

**Review by nialexsan — APPROVED** [review]

**Pull request merged**